app/javascript/components/HomePageContent.js

```diff
@@ -29,7 +29,7 @@ const LinkableSearchTabs = function(props) {
   const showGenesTab = location.pathname.startsWith('/single_cell/app/genes')
   return (
     <div>
-      <nav className="nav search-links">
+      <nav className="nav search-links" data-tablist-name={'search'} role="tablist">
```

Comment:
this is a fixed string, so it doesn't need to be a variable
```suggestion
      <nav className="nav search-links" data-tablist-name="search" role="tablist">
```